seacrowd/sea_datasets/thai_ser/thai_ser.py

```diff
@@ -0,0 +1,200 @@
+    "emotion_label": "https://github.com/vistec-AI/dataset-releases/releases/download/v1/emotion_label.json",
+    "studio": {
+        "studio1-10": "https://github.com/vistec-AI/dataset-releases/releases/download/v1/studio1-10.zip",
+        "studio11-20": "https://github.com/vistec-AI/dataset-releases/releases/download/v1/studio11-20.zip",
+        "studio21-30": "https://github.com/vistec-AI/dataset-releases/releases/download/v1/studio21-30.zip",
+        "studio31-40": "https://github.com/vistec-AI/dataset-releases/releases/download/v1/studio31-40.zip",
+        "studio41-50": "https://github.com/vistec-AI/dataset-releases/releases/download/v1/studio41-50.zip",
+        "studio51-60": "https://github.com/vistec-AI/dataset-releases/releases/download/v1/studio51-60.zip",
+        "studio61-70": "https://github.com/vistec-AI/dataset-releases/releases/download/v1/studio61-70.zip",
+        "studio71-80": "https://github.com/vistec-AI/dataset-releases/releases/download/v1/studio71-80.zip",
+    },
+    "zoom": {"zoom1-10": "https://github.com/vistec-AI/dataset-releases/releases/download/v1/zoom1-10.zip", "zoom11-20": "https://github.com/vistec-AI/dataset-releases/releases/download/v1/zoom11-20.zip"},
+}
+
+_SUPPORTED_TASKS = [Tasks.SPEECH_EMOTION_RECOGNITION]
+
+_SOURCE_VERSION = "1.0.0"
+_SEACROWD_VERSION = "1.0.0"
+
+
+class ThaiSER(datasets.GeneratorBasedBuilder):
+    """Thai speech emotion recognition dataset THAI SER contains 100 recordings (80 studios and 20 zooms)."""
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    SEACROWD_SCHEMA_NAME = "speech"
+    _LABELS = ["Neutral", "Angry", "Happy", "Sad", "Frustrated"]
+
+    BUILDER_CONFIGS = [
+        # zoom
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_source",
+            version=SOURCE_VERSION,
+            description=f"{_DATASETNAME} source schema",
+            schema="source",
+            subset_id=f"{_DATASETNAME}_zoom",
+        ),
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_seacrowd_{SEACROWD_SCHEMA_NAME}",
+            version=SEACROWD_VERSION,
+            description=f"{_DATASETNAME} SEACrowd schema",
+            schema=f"seacrowd_{SEACROWD_SCHEMA_NAME}",
+            subset_id=f"{_DATASETNAME}_zoom",
+        ),
+        # studio
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_studio_source",
+            version=SOURCE_VERSION,
+            description=f"{_DATASETNAME} source schema",
+            schema="source",
+            subset_id=f"{_DATASETNAME}_studio",
+        ),
+        SEACrowdConfig(
+            name=f"{_DATASETNAME}_studio_seacrowd_{SEACROWD_SCHEMA_NAME}",
+            version=SEACROWD_VERSION,
+            description=f"{_DATASETNAME} SEACrowd schema",
+            schema=f"seacrowd_{SEACROWD_SCHEMA_NAME}",
+            subset_id=f"{_DATASETNAME}_studio",
+        ),
```

Comment:
I do not think that we should split this as subsets.
Based on the publication, it seems to be either with or without zoom
(https://github.com/vistec-AI/vistec-ser/blob/main/vistec_ser/examples/train_aisser.py#L17-L18)

---

IMHO, we have 3 options:
1. Follow the original splits of {studio, studio+zoom}
2. No splitting (single subset), as there is explicit indication of subsetting the dataset in the published writings.
3. Splits based on Recording Environments, i.e. {StudioA, StudioB, Zoom}, or even further splitting based on session types.

---

However, I am not an expert in this SER area nor able to speak Thai,
so please correct me if I'm wrong (CMIIW).
cc: @holylovenia

Reply:
@zwenyu thanks for the updates on the other parts, LGTM.
As for this part, for the time being, could you please follow the split from the original?
i.e. normal and `--include-zoom`

Reply:
Updated to studio, and studio and zoom splits